about.css

```diff
@@ -0,0 +1,465 @@
+  cursor: pointer;
+}
+
+.header .mobile-nav .a:hover,
+.header .mobile-nav .closeBtn:hover {
+  color: #ec5242;
+}
+
+.header .mobile-nav .li-cc {
+  margin-top: 30px;
+}
+
+.header .mobile-nav .a-cc {
+  border: solid 4px red;
+  padding: 10px;
+  color: #ec5242;
+  white-space: nowrap;
+}
+
+.header .desktop-nav {
+  display: none;
+}
+
+/* Section 1 */
+
+.section-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  margin: 60px 0;
+  text-align: center;
+}
+
+.section-1 .hello {
+  color: #ec5242;
+  font-size: larger;
+}
+
+.section-1 .h1 {
+  color: #ec5242;
+  font-size: 40px;
+  line-height: 40px;
+  margin-bottom: 20px;
+}
+
+.section-1 .p {
+  background: white;
+  border: 1px solid #d3d3d3;
+  padding: 20px;
+  line-height: 25px;
+}
+
+.section-1 .desktop-p {
+  display: none;
+}
+
+.section-1 .email {
+  color: #333;
+  text-decoration: underline;
+  text-underline-position: under;
+  padding-bottom: 5px;
+  font-size: larger;
+  font-weight: 500;
+  margin-top: -10px;
+}
+
+/* Section 2 */
+
+.section-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 50px;
+  padding: 50px 5vw;
+  text-align: center;
+}
+
+.section-2 .text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+  padding: 0 30px;
+}
+
+.section-2 .red-line {
+  height: 1px;
+  width: 25px;
+  background: #ec5242;
+}
+
+.section-2 .desktop-p {
+  display: none;
+}
+
+.section-2 .logo {
+  max-height: 114px;
+  max-width: 270px;
+  border: 1px solid #d3d3d3;
+  padding: 30px 60px;
+}
+
+.section-2 .grey-line {
+  height: 1px;
+  width: 100vw;
+  background: #d3d3d3;
+}
+
+.section-2 .event-container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.section-2 .event {
+  height: 150px;
+  width: 300px;
+}
+
+.section-2 .event .h3 {
+  font-size: 30px;
+  font-weight: 900;
+  color: white;
+}
+
+.section-2 .event .p {
+  font-size: 16px;
+  font-weight: 900;
+  color: rgb(218, 216, 216);
+}
+
+.section-2 .event-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  background:
+    linear-gradient(
+      0deg,
+      rgba(232, 117, 107, 0.687),
+      rgba(233, 74, 74, 0.664)
+    ),
+    url('images/pexels-matheus-bertelli-3321797.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.section-2 .event-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  background:
+    linear-gradient(
+      0deg,
+      rgba(232, 117, 107, 0.687),
+      rgba(233, 74, 74, 0.664)
+    ),
+    url('images/pexels-matheus-bertelli-3321802.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* Footer Section */
+
+.footer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  padding: 50px 5vw;
+  background: #272a31;
+  color: #9b9b9b;
+}
```

Comment:
- [ ] It can't get closer than this! but the background color of your footer in the mobile version is different from the original design and as a frontend developer, you should pay attention to small details like this as it's a quality of a great developer.
- Your design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/bc003d9d-80c7-475f-8867-63f6be25af58)
- Original design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/703a8e1f-3d62-4378-8ed2-2b4f516ddfa6)